css/_uswds-theme-custom-styles.scss

```diff
@@ -1,3 +1,5 @@
+@use "sass:math";
```

Comment:
Was this called out in a compile warning or lint error?

It's odd that `map-merge`, `nth`,  wasn't flagged.

Reply:
Some history: This update came when I pulled in the [package test PR](https://github.com/uswds/uswds-site/pull/1487/files) for debugging. I left it in because it cleared out the deprecation warnings when I built the sass. Let me know if you want me to revert until we can do a more comprehensive look at the math operator.

Reply:
No problems with the use of `math.div()`. Moreso that our linter or the compiler didn't flag `map-merge → map.merge()`.